tests/integration/test_backup_restore_on_cluster/test_concurrency.py

```diff
@@ -81,6 +81,11 @@ def create_and_fill_table():
         ") ENGINE=ReplicatedMergeTree('/clickhouse/tables/tbl/', '{replica}')"
         "ORDER BY tuple()"
     )
+    assert_eq_with_retry(
+        node0,
+        "SELECT count() from clusterAllReplicas('cluster', system.tables) where table='tbl' and database='default'",
+        TSV([num_nodes]),
+    )
     for i in range(num_nodes):
         nodes[i].query(f"INSERT INTO tbl VALUES ({i})")
```

Comment:
And that's because `num_nodes` in `test_disallow_concurrency.py` was changed to 4 in https://github.com/ClickHouse/ClickHouse/pull/47216. So the flaky check failure was actually related to the changes.